scalableminds/webknossos
# Fix context menu overlay being to high

In the current version of the master, the invisible overlay behind the node context menu is too high. It does not consider the navbar height.

### URL of deployed dev instance (used for testing):
- <https://fixcontextmenuoverlay.webknossos.xyz>

### Steps to test:
- Open a hybrid / skeleton tracing.
- Use `shift + right click` to open the context menu. No scroll bars should appear at the bottom and at the right side of the page.
- Do the same thing on the master. In this case, scroll bars should appear.

### Issues:
- No issue for this PR.
------
